cmd/sonicd/app/launcher.go

```diff
@@ -222,6 +222,10 @@ func lachesisMain(ctx *cli.Context) error {
 	}
 
 	metrics.SetDataDir(cfg.Node.DataDir) // report disk space usage into metrics
+	if ctx.GlobalIsSet(config.FakeNetFlag.Name) {
```

Comment:
As in #334, I do not think it is a good idea to link the cache configuration with the fake-net flag. This would prohibit scalability tests on fake-nets.